impeller/renderer/context.h

```diff
@@ -181,9 +181,7 @@ class Context {
   ///             pending work.
   virtual void SetSyncPresentation(bool value) {}
 
-  //----------------------------------------------------------------------------
-  /// @brief Accessor for a pool of HostBuffers.
-  Pool<HostBuffer>& GetHostBufferPool() const { return host_buffer_pool_; }
+  virtual const std::shared_ptr<HostBuffer> GetTransientsBuffer() const = 0;
```

Comment:
Would it be problematic to track the transients buffer in the `ContentContext` and trigger per-frame lifecycle stuff from `EntityPass::Render()` (called once per frame, even when fancy `Picture` stuff is afoot)?

Reply:
I think ContentContext would be a reasonable place to put this.